content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -152,6 +152,40 @@ kubectl delete -k manifests/kustomize/env/dev
 # kubectl delete -k manifests/kustomize/env/gcp
 ```
 
+## Best practices maintaining custom manifests
+
+### Maintain a repo for your manifests
+
+Save the following to a source controlled repo.
+
+File `kustomization.yaml`.
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Edit the following to change the deployment to your custom namespace.
+namespace: kubeflow
+# You can add other customizations here using kustomize.
+# Edit ref in the following link to deploy a different version of Kubeflow Pipelines.
+bases:
+- github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref={{% kfp-latest-version %}}
+```
+
+### How to
+* Deploy: `kubectl apply -k $YOUR_REPO`
+* Upgrade:
+    1. (Recommended) backup your data storages for KFP.
+    1. Edit `ref={{% kfp-latest-version %}}` to a version you want to upgrade to.
+￼    Check [Kubeflow Pipelines github repo](https://github.com/kubeflow/pipelines/releases) for available releases.
+￼    1. Deploy: `kubectl apply -k $YOUR_REPO`.
+￼* Downgrade:
```

Comment:
I don't think we support downgrade as it can causes uncertain behavior. e.g. new version api that @jingzhang36 is working on.

Reply:
Note I mentioned making a backup and first restoring PVs to backed up state before downgrade.
I think this is doable, but I need to actually try it and prepare detailed instructions.

I now change my mind and prefer skipping downgrade instructions for now.
What do you think?

Reply:
sg